module/x/auction/keeper/invariants.go

```diff
@@ -0,0 +1,75 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/Gravity-Bridge/Gravity-Bridge/module/x/auction/types"
+)
+
+/*	Auction Module Invariants
+	For background: https://docs.cosmos.network/main/building-modules/invariants
+	Invariants on Gravity Bridge chain will be enforced by most validators every 200 blocks, see module/cmd/root.go for
+	the automatic configuration. These settings are overrideable and not consensus breaking so there are no firm
+	guarantees of invariant checking no matter what is put here.
+*/
+
+// AllInvariants collects any defined invariants below
+func AllInvariants(k Keeper) sdk.Invariant {
+	return func(ctx sdk.Context) (string, bool) {
+		res, stop := ModuleBalanceInvariant(k)(ctx)
```

Comment:
shouldn't we do a store validity invariant that just reads all the auctions in the store? But I guess that already happens here, is there anything else we could read?

Reply:
I added one to go through the auctions and check the denoms aren't the native token